echopype/echodata/echodata.py

```diff
@@ -124,15 +124,16 @@ def _load_file(self, raw_path):
         """Lazy load Top-level, Beam, Environment, and Vendor groups from raw file."""
         for group, value in self.__group_map.items():
             # EK80 data may have a Beam_power group if both complex and power data exist.
+            # ADCP data adds a Beam_complex group
+            ds = None
             try:
                 ds = self._load_group(
                     raw_path,
                     group=value["ep_group"],
                 )
             except (OSError, GroupNotFoundError):
-                if group == "beam_power":
-                    ds = None
-                    pass
+                # Skips group not found errors for EK80 and ADCP
```

Comment:
@leewujung This was needed in order for echodata to ignore the missing group since the test only uses ek60 data!